crypto/rand/drbg_lib.c

```diff
@@ -127,7 +127,7 @@ int RAND_DRBG_instantiate(RAND_DRBG *drbg,
     unsigned char *nonce = NULL, *entropy = NULL;
     size_t noncelen = 0, entlen = 0;
```

Comment:
`entroylen` would be more consistently named.

Reply:
I already considered changing the name some time ago, because I hate inconsistent naming, too. But when I saw all the cryptic 'ent' names in drbgtest.c, I hesitated:
```
test/drbgtest.c:168:    t.entlen = td->entlen;
test/drbgtest.c:181:    t.entlen = td->entreseedlen;
test/drbgtest.c:199:    t.entlen = td->entlen_pr;
test/drbgtest.c:212:    t.entlen = td->entprlen_pr;
test/drbgtest.c:222:    t.entlen = td->entglen_pr;
```
So thanks for encouraging me to straighten out these names, too ;-)

Reply:
Thanks.  That's even better.